cli/analytics.go

```diff
@@ -49,12 +50,12 @@ func redactStepInputs(environment map[string]string, inputs []models.Environment
 func redactWithSecrets(inputValue string, secrets []string) (string, error) {
 	src := bytes.NewReader([]byte(inputValue))
 	dstBuf := new(bytes.Buffer)
-	secretFilterDst := filterwriter.New(secrets, dstBuf)
+	redactWriterDst := redactwriter.New(secrets, dstBuf, log.NewLogger())
```

Comment:
Could `log.NewLogger()` cause any issues with the structured logging? Can no recall if there are guidelines about it.

Reply:
Ahh damn I do not know what I was thinking. Yes, the structured logger needs to be used instead of this one.

This struct requires the logger from the go-utils package but for the structured logging a different one needs to be used. We already had an adapter for this which was used for the analytics. I have made that one public and used it instead.